docs/guides/using-organizations.md

```diff
@@ -0,0 +1,104 @@
+# Using Organizations (Alpha)
```

Comment:
Our feature stages doc suggests that our alpha features are not hidden behind an experiment? 

Maybe unrelated, but if we enable alpha features by default, should they be after experimental? Or is it a parallel category?

Either way, this being "alpha" seems contradictory.

https://coder.com/docs/contributing/feature-stages#alpha-features

Reply:
Oh I see, the page itself is in Alpha.

Reply:
Will fix